pygenometracks/tracks/LinksTrack.py

```diff
@@ -98,12 +121,55 @@ def __init__(self, *args, **kwarg):
         file_h.close()
         self.interval_tree = interval_tree
 
-        if 'color' not in self.properties:
-            self.properties['color'] = 'blue'
+        if 'line width' not in self.properties and not has_score:
+            self.log.warning("*WARNING* for section {}"
+                             " no line width has been set but some "
+                             "lines do not have scores."
+                             "line width has been set to "
+                             "0.5".format(self.properties['section_name']))
+            self.properties['line width'] = 0.5
 
         if 'alpha' not in self.properties:
             self.properties['alpha'] = 0.8
 
+        if 'color' not in self.properties:
+            self.properties['color'] = DEFAULT_LINKS_COLOR
+        self.colormap = None
+        # check if the color given is a color map
+        if not matplotlib.colors.is_color_like(self.properties['color']):
```

Comment:
That entire color handling looks like a nice utility function that could be reused and maybe moved out of this python script.
But this can happen later ...

Reply:
Yes, you are right it is also used more or less in the bedTrack. I should make a utility but if you don't mind I would do it when both #114 and #116 are merged. It will reduce the conflicts.

Reply:
Agreed!

Reply:
See #119